package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
- [ ] Kindly add `react-router-dom` to the dependencies. without this dependency, the project does not work. 

😄 👍